alpha/alpha-server/src/test/java/org/apache/servicecomb/pack/alpha/server/AlphaIntegrationTest.java

```diff
@@ -286,7 +286,7 @@ public void doNotCompensateDuplicateTxOnFailure() {
     blockingStub.onTxEvent(someGrpcEvent(TxAbortedEvent));
     await().atMost(1, SECONDS).until(() -> receivedCommandsCounter.get() > 1);
```

Comment:
The test failure is caused by the receivedCommandCounter is less than 2. 
I think we should wait for a bit more time for it.

Reply:
L287 passed, Fails on L289

Reply:
await() just wait for the condition is true or the timeout.

Reply:
hasItems cannot assert we got all the event we wants.

Reply:
Ok, I will try to use containsInAnyOrder instead of contains

Reply:
done